src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIntegrationTestLegacy.cs

```diff
@@ -133,6 +152,18 @@ public async Task Build_DoesNotPrintsWarnings_IfProjectFileContainsRazorFiles()
             }
         }
 
+        [Fact]
+        public async Task Publish_IncludesRefAssemblies_WhenCopyRefAssembliesToPublishDirectoryIsSet()
+        {
+            using (CreateTestProject())
+            {
+                var result = await DotnetMSBuild("Publish", "/p:CopyRefAssembliesToPublishDirectory=true");
```

Comment:
If it's on by default why do we need to specify `CopyRefAssembliesToPublishDirectory`?

Reply:
`CopyRefAssembliesToPublishDirectory` is off by default in all cases. `PreserveCompilationReferences` is on for build in 2.x projects, and set to false unless `CopyRefAssembliesToPublishDirectory` is set.